hummingbot/connector/exchange/hitbtc/hitbtc_constants.py

```diff
@@ -0,0 +1,59 @@
+# A single source of truth for constant variables related to the exchange
+class Constants:
+    EXCHANGE_NAME = "hitbtc"
+    REST_URL = "https://api.hitbtc.com/api/2"
+    REST_URL_AUTH = "/api/2"
+    # WS_PRIVATE_URL = "wss://stream.crypto.com/v2/user"
+    WS_PRIVATE_URL = "wss://api.hitbtc.com/api/2/ws/trading"
+    # WS_PUBLIC_URL = "wss://stream.crypto.com/v2/market"
+    WS_PUBLIC_URL = "wss://api.hitbtc.com/api/2/ws/public"
+
+    HBOT_BROKER_ID = "refzzz48"
```

Comment:
@dennisocana Is this the right `HBOT_BROKER_ID` for HitBTC?

Reply:
@CrimsonJacket yes, this is correct 👍